docs/application/dotnet/guides/nui/texteditor.md

```diff
@@ -0,0 +1,113 @@
+| `DecorationBoundingBox`            | Rectangle   | The decorations (handles etc) will positioned within this area on-screen |
+| `EnableMarkup`                     | Boolean     | Whether the mark-up processing is enabled |
+| `InputColor`                       | Vector4     | The color of the new input text          |
+| `InputFontFamily`                  | String      | The font's family of the new input text  |
+| `InputFontStyle`                   | PropertyMap | The font's style of the new input text   |
+| `InputPointSize`                   | Float       | The font's size of the new input text in points |
+| `LineSpacing`                      | Float       | The default extra space between lines in points |
+| `InputLineSpacing`                 | Float       | The extra space between lines in points  |
+| `Underline`                        | PropertyMap | The default underline parameters         |
+| `InputUnderline`                   | String      | The underline parameters of the new input text |
+| `Shadow`                           | PropertyMap | The default shadow parameters            |
+| `InputShadow`                      | String      | The shadow parameters of the new input text |
+| `Emboss`                           | String      | The default emboss parameters            |
+| `InputEmboss`                      | String      | The emboss parameters of the new input text |
+| `Outline`                          | PropertyMap | The default outline parameters           |
+| `InputOutline`                     | String      | The outline parameters of the new input text |
+| `SmoothScroll`                     | Boolean     | Enable or disable the smooth scroll animation |
+| `SmoothScrollDuration`             | Float       | Sets the duration of smooth scroll animation |
+| `EnableScrollBar`                  | Boolean     | Enable or disable the scroll bar         |
+| `ScrollBarShowDuration`            | Float       | Sets the duration of scroll bar to show  |
+| `ScrollBarFadeDuration`            | Float       | Sets the duration of scroll bar to fade out |
+| `PixelSize`                        | Float       | The size of font in pixels               |
+| `LineCount`                        | Integer     | The line count of text                   |
+| `EnableSelection`                  | Boolean     | Enables Text selection                   |
+| `Placeholder`                      | PropertyMap | Sets the placeholder : text, color, font family, font style, point size, and pixel size |
+| `LineWrapMode`                     | LineWrapMode | Line wrap mode when text lines are greater than the layout width |
+| `TranslatablePlaceholderText`      | String      | The TranslatablePlaceholderText property. The text can set the SID value. |
+| `TranslatableText`                 | String      | The TranslatableText property. The text can set the SID value. |
+
```

Comment:
Add Period at the end of all the above sentence.
For example: 
Outline`                          | PropertyMap | The default outline parameters.           |

Reply:
Done.